lib/MetaProcessor/MetaParser.cpp

```diff
@@ -474,6 +474,34 @@ namespace cling {
     return false;
   }
 
+  // dumps/creates a trace of the requested representation.
+  bool MetaParser::istraceCommand() {
+    if (getCurTok().is(tok::ident) &&
+        getCurTok().getIdent().equals("trace")) {
+      consumeToken();
+      skipWhitespace();
+      if (!getCurTok().is(tok::ident))
+          return false;
+      std::string ident = getCurTok().getIdent();
+      consumeToken();
+      skipWhitespace();
+      if (getCurTok().is(tok::eof)) {
+        m_Actions->actOnstatsCommand(ident=="ast" 
+            ? llvm::StringRef("asttree") : llvm::StringRef(ident), llvm::StringRef());
```

Comment:
isn't the "t" in "ast" already the "tree"? :-) I'd suggest "<complete>" (no identifier can be called `<complete>`) if that is what it means?